windows-agent/cmd/ubuntu-pro-agent/agent/agent_test.go

```diff
@@ -181,6 +182,50 @@ func TestAppGetRootCmd(t *testing.T) {
 	require.NotNil(t, a.RootCmd(), "Returns root command")
 }
 
+func TestPublicDir(t *testing.T) {
+	// Not parallel because we modify the environment
+
+	testCases := map[string]struct {
+		emptyEnv bool
+		badPath  bool
+
+		wantErr bool
+	}{
+		"Success providing a public directory":               {},
+		"Error when %UserProfile% is empty":                  {emptyEnv: true, wantErr: true},
```

Comment:
I know there are 3 cases but still separating the success/error cases by a blank line sounds like a better way if we add future parameters to it :)

Reply:
Yep, my bad.